recipes/pyupgrade/LICENSE

```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2017 Anthony Sottile
```

Comment:
Let's make sure this is submitted upstream.

Reply:
Done: https://github.com/asottile/pyupgrade/pull/90